backend/src/sms/middlewares/sms-callback.middleware.ts

```diff
@@ -34,6 +34,23 @@ const isAuthenticated = (
   return res.sendStatus(403)
 }
 
+const isAuthenticatedTransactional = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Response | void => {
+  if (!req.get('authorization')) {
+    res.set('WWW-Authenticate', 'Basic realm="Twilio"')
+    return res.sendStatus(401)
+  }
+  if (
```

Comment:
**[nit]** I think it's clearer to use guard clauses to handle all the 4xx cases and leave the happy path at the end

Reply:
have updated, take a look

Reply:
lgtm thanks!